docs/common/community/talks.md

```diff
@@ -25,6 +25,12 @@ Here is a list of talks and other videos related to Iceberg.
 ### [Expert Roundtable: The Future of Metadata After Hive Metastore](https://www.youtube.com/watch?v=7_Pt1g2x-XE)
 **Date**: November 15, 2021, **Authors**: Lior Ebel, Seshu Adunuthula, Ryan Blue & Oz Katz
 
+### [Presto and Apache Iceberg: Building out Modern Open Data Lakes](https://www.youtube.com/watch?v=OJQHVPChYHw)
+**Date**: November 10, 2021, **Authors**: Daniel Weeks, Chunxu Tang
```

Comment:
I noticed the youtube video was posted on Dec 21st so is Nov 10th the date the actual talk took place? I think either one is fine but thought I would ask.